katalog/nginx/bases/controller/DaemonSet-ingress-nginx-controller.yml

```diff
@@ -37,27 +55,26 @@ spec:
                   - /wait-shutdown
           args:
             - /nginx-ingress-controller
-            - --configmap=$(POD_NAMESPACE)/nginx-configuration
             - --publish-service=$(POD_NAMESPACE)/ingress-nginx
-            - --annotations-prefix=nginx.ingress.kubernetes.io
-            - --http-port=8080
-            - --https-port=8443
-            - --ingress-class=nginx
             - --election-id=ingress-controller-leader
             - --controller-class=k8s.io/ingress-nginx
+            - --ingress-class=nginx
+            - --configmap=$(POD_NAMESPACE)/ingress-nginx-controller
             - --validating-webhook=:9443
-            - --validating-webhook-certificate=/usr/local/certificates/tls.crt
-            - --validating-webhook-key=/usr/local/certificates/tls.key
+            - --validating-webhook-certificate=/usr/local/certificates/cert
+            - --validating-webhook-key=/usr/local/certificates/key
```

Comment:
beware that this change in the order of the arguments breaks several kustomize patches that uses the arguments index to replace the argument. Both kustomize patches in the module (dual-nginx) as in the distro. If we can I'd keep the same ordering as before to minimise the impact.

Reply:
That's true, but I think we can do this "leap" for once (it's a major release, after all) and then stick with the upstream will come easier in the future.